tests/integrations/mcs/scheduling/api_test.go

```diff
@@ -513,6 +513,15 @@ func (suite *apiTestSuite) checkFollowerForward(cluster *tests.TestCluster) {
 	defer cancel()
 	follower, err := cluster.JoinAPIServer(ctx)
 	re.NoError(err)
+	defer func() {
+		leader := cluster.GetLeaderServer()
+		cli := leader.GetEtcdClient()
+		_, err := cli.MemberRemove(ctx, follower.GetServer().GetMember().ID())
```

Comment:
Should we remove the follower from `cluster.servers`, destroy the follower, and wait for Leader?